stream/src/public_key/handshake.rs

```diff
@@ -43,13 +43,13 @@ pub fn create_handshake<C: Scheme>(
 }
 
 pub struct Handshake {
-    pub(super) ephemeral_public_key: x25519_dalek::PublicKey,
```

Comment:
I always hated that I used `(super)` here. Now that `Handshake` is entirely private, we can remove all of this.